### PR TITLE
KPrototypes Part 1 + Misc

### PR DESCRIPTION
**Pull request by oualib**
- Supporting KPrototypes except to_python / memmodel convertion
 - changing the header (2023)
 - adding the right vertica-python version

TODO: unit test + memmodel convertion

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
```
Comment (afard) [inline setup.py:19]: Why don't you update the copyright on this file?

File: verticapy/learn/vmodel.py [hunk 2]
```diff
@@ -2502,7 +2535,9 @@ def to_python(
             )
             func += "\treturn np.apply_along_axis(predict_tree_final, 1, X)\n"
             return func
-        elif self.type in ("NearestCentroid", "KMeans"):
+        elif self.type in ("KPrototypes",):
+            pass # TODO
```
Comment (afard) [inline verticapy/learn/vmodel.py:2539]: Are you planning to implement this before merge?

**Comment by afard** [comment]
Where are the test of KPrototypes?

**Comment by oualib** [comment]
> Where are the tests of KPrototypes?

It will be done in the PART2 as it needs all the different functions and some of them are not yet implemented.

**Review by afard — APPROVED** [review]
Let's make adding the tests of KPrototypes a high priority.

**Pull request merged**